preprocessing/nextclade/src/loculus_preprocessing/prepro.py

```diff
@@ -570,8 +570,7 @@ def upload_flatfiles(processed: Sequence[SubmissionData], config: Config) -> Non
                     processedFields=[
                         AnnotationSource(name="embl_upload", type=AnnotationSourceType.METADATA)
                     ],
-                    message="Failed to create or upload EMBL file "
-                    "please contact your administrator.",
+                    message="Failed to create or upload EMBL file. Please contact your administrator.",
```

Comment:
would be nice to keep the lines shorter

Reply:
Ah I see `line-length = 100` (too low for my taste but ok). And yes, the CI should enforce this otherwise there will be more and more violations.